common/build.gradle.kts

```diff
@@ -0,0 +1,38 @@
+plugins {
+    alias(libs.plugins.android.library)
+    alias(libs.plugins.kotlin.android)
+}
```

Comment:
Please re-analyse whether your `common` module needs to be an `android` or `pure Java/Kotlin` module.

Reply:
sure